app/javascript/css/forms.scss

```diff
@@ -74,6 +74,22 @@ select,
   }
 }
 
+input[type="checkbox"] {
+  &:active,
+  &:focus {
+  outline: 1px solid var(--grey-150);
+  }
+}
+
+.focus {
+  &:active,
+  &:focus,
+  &:focus-within {
+  outline: -webkit-focus-ring-color auto 1px;
```

Comment:
I am fine with using this or getting the hsl value for it, unless we would prefer to use the grey-150 for focus on it

Reply:
fine using the default for this